HISTORY.md

```diff
@@ -1,3 +1,7 @@
+v2.2.3
```

Comment:
This should be a minor bump, not patch.

Feel free to leave version bumps out of PRs, we can sort this during release.

Reply:
Sounds good. Reverted for now. When we can have the new release, our product GA needs this fix. :)